joint_limits:
  shoulder_lift_joint:
    has_velocity_limits: true
    max_velocity: 6.97
    has_acceleration_limits: true
    max_acceleration: 1.0
  shoulder_pan_joint:
    has_velocity_limits: true
    max_velocity: 3.48
    has_acceleration_limits: true
    max_acceleration: 1.0
  elbow_joint:
    has_velocity_limits: true
    max_velocity: 2.96
    has_acceleration_limits: true
    max_acceleration: 1.0
  wrist_1_joint:
    has_velocity_limits: true
    max_velocity: 2.26
    has_acceleration_limits: true
    max_acceleration: 1.0
  wrist_2_joint:
    has_velocity_limits: true
    max_velocity: 2.26
    has_acceleration_limits: true
    max_acceleration: 1.0
  wrist_3_joint:
    has_velocity_limits: true
    max_velocity: 2.96
    has_acceleration_limits: true
    max_acceleration: 1.0
  endo_first_joint:
    has_velocity_limits: true
    max_velocity: 3.48
    has_acceleration_limits: true
    max_acceleration: 1.0
  endo_second_joint_first_dof:
    has_velocity_limits: true
    max_velocity: 3.48
    has_acceleration_limits: true
    max_acceleration: 1.0
  endo_second_joint_second_dof:
    has_velocity_limits: true
    max_velocity: 3.48
    has_acceleration_limits: true
    max_acceleration: 1.0
